src/force-graph.js

```diff
@@ -148,9 +148,11 @@ export default Kapsule({
     onNodeDrag: { default: () => {}, triggerUpdate: false },
     onNodeDragEnd: { default: () => {}, triggerUpdate: false },
     onNodeClick: { default: () => {}, triggerUpdate: false },
+    onNodeDoubleClick: { default: () => {}, triggerUpdate: false },
```

Comment:
These new props don't seem to have any effect. Do you still intend to include them?

Reply:
Good catch, should be removed since double clicks are probably best handled externally.